src/views/pages/Login/index.vue

```diff
@@ -24,17 +24,24 @@ import Vue from 'vue';
 import TokenService from '@/services/Token';
 import CoreAPI from '@/utils/CoreAPI';
 import SyncService from '@/services/Sync';
+import { MutationType as UserMutation } from '@/store/modules/user/types';
 
 export default Vue.extend({
   name: 'Login',
   mixins: [TokenService.tokenMixin()],
   methods: {
     async login() {
       try {
-        await CoreAPI.login(this.credentials);
+        const { uuid, jwt, type } = await CoreAPI.login(this.credentials);
+        this.$store.commit(UserMutation.SET_USER_ID, { userId: uuid });
```

Comment:
I think it'd be better to be specific that this is a uuid, not just an ID. I also think it'd better to just call the field uuid instead of userId, since access from the store would be 
```
this.$store.state.user.uuid
```
and 
```
this.$store.state.user.userId
```

is kinda redundant.